### ISSUE TITLE
[css-inline] Specify fallback baselines estimations for text?

### ISSUE THREAD

**Issue by yiyix** [w3c/csswg-drafts#9232]
In the [CSS spec](https://drafts.csswg.org/css-inline/#hanging-baseline), hanging baseline is defined as the following `Corresponds to hanging baseline from which characters in Tibetan and similar unicameral scripts with a strong but not absolute top edge seem to “hang”. Assigned to hang in OpenType and to bsln value 3 in TrueType AAT.`

It didn't describe what's the fallback value for hanging baseline if "hang" is missing in OpenTyoe table.  The definition is not entirely clear as well, it only says `a strong but not absolute top edge`. So it allows browsers to have their own interpretation of the baseline value if "hang" is missing in OpenType table. I am wondering if we could add a note to suggest how this value could be computed from font ascent and font descent when it's missing.

[`ideographic baseline`](https://drafts.csswg.org/css-inline/#ref-for-ideographic-under-baseline) has the same issue.

**Comment by drott** [comment]
I agree this should be addressed. UA's implementing different heuristics when the `hang` entry is not available from the `base` OpenType table is not interoperable.

**Comment by drott** [comment]
CC @kojiishi
